sway/desktop/output.c

```diff
@@ -1028,10 +1028,10 @@ void handle_output_power_manager_set_mode(struct wl_listener *listener,
 	struct output_config *oc = new_output_config(output->wlr_output->name);
 	switch (event->mode) {
```

Comment:
Could be simplified in `oc->power = event->mode` but it'll lose some readability.

Reply:
I'd rather not rely on the values meaning the same in the protocol and in the Sway codebase.